config/v1/types_infrastructure.go

```diff
@@ -705,7 +705,45 @@ type AlibabaCloudResourceTag struct {
 
 // NutanixPlatformSpec holds the desired state of the Nutanix infrastructure provider.
 // This only includes fields that can be modified in the cluster.
-type NutanixPlatformSpec struct{}
+type NutanixPlatformSpec struct {
+	// prismCentral holds the endpoint address and port to access the Nutanix Prism Central
+	// +kubebuilder:validation:Required
+	PrismCentral NutanixPrismEndpoint `json:"prismCentral"`
```

Comment:
When a proxy is configured, is this endpoint accessed via the proxy or not accessed via the proxy?

Reply:
Nutanix Prism provides proxy support and it's documented here https://portal.nutanix.com/page/documents/details?targetId=Prism-Central-Guide-Prism-v6_0:mul-http-proxy-configure-pc-t.html.

Reply:
Added the comment.